packages/conventional-changelog-core/lib/merge-config.js

```diff
@@ -32,6 +32,39 @@ function semverTagsPromise(options) {
   });
 }
 
+/**
+ * Generates a compare url on the respective repository host
+ *
+ * @param {any} context the same context object as the one that's used in the templates
+ * @returns {string} compare url for gitlab | github | bitbucket
+ */
+function generateCompareUrl(context) {
+  var generatedUrl = '';
```

Comment:
Could you switch to `let` over `var`, and switch to template literals over single quotes?